Content.Server/Crayon/CrayonSystem.cs

```diff
@@ -134,7 +136,7 @@ private void OnCrayonInit(EntityUid uid, CrayonComponent component, ComponentIni
     private void OnCrayonDropped(EntityUid uid, CrayonComponent component, DroppedEvent args)
     {
         if (TryComp<ActorComponent>(args.User, out var actor))
-            component.UserInterface?.Close(actor.PlayerSession);
+            _uiSystem.CloseUi(component.UserInterface!, actor!.PlayerSession);
```

Comment:
This null handling looks very suss, actor.PlayerSession is never null and component.UserInterface can be null.